tools/benchmark

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+LEGOFY=$(pwd)/bin/legofy;
+PATH_IMAGES=$(pwd)/images/benchmark/;
```

Comment:
I'd rather have these relative to script path, using:

```sh
readonly PROGDIR=$(dirname "$0")
LEGOFY=$PROGDIR/../bin/legofy
PATH_IMAGES=$PROGDIR/../images/benchmark
```

Also semocolons are optional here. Also I'd rather not include the trailing `/` to be consistent with `dirname` output and make variable concatenation lower more readable.